timetracker/entries/views.py

```diff
@@ -111,3 +88,10 @@ def project_detail(request, pk):
         'project': project,
         'form': form,
     })
+
+
+class RootRedirectView(RedirectView):
+    permanent = False
+
+    def get_redirect_url(self):
```

Comment:
You can use `redirect_url = reverse_lazy('client-list')` instead :wink: (`reverse_lazy` is at the same place as `reverse`)